components/database/libpqxx/Makefile

```diff
@@ -11,20 +11,20 @@
 
 #
 # Copyright 2017 Alexander Pyhalov
+# Copyright 2018 Michal Nowak
 #
 
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= libpqxx
-COMPONENT_VERSION= 4.0.1
-COMPONENT_REVISION= 3
+COMPONENT_VERSION= 6.2.4
 COMPONENT_SUMMARY= C++ binding library for PostgreSQL
 COMPONENT_SRC= $(COMPONENT_NAME)-$(COMPONENT_VERSION)
-COMPONENT_ARCHIVE= $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE= $(COMPONENT_VERSION).tar.gz
```

Comment:
Please, store archive as libpqxx-6.2.4.tar.gz. not 6.2.4.tar.gz. To do this, set desired COMPONENT_ARCHIVE, but use $(COMPONENT_VERSION).tar.gz in COMPONENT_ARCHIVE_URL.